src/Storage/TSDatabaseView.h

```diff
@@ -54,4 +54,7 @@ extern NSString *const TSSecondaryDevicesDatabaseViewExtensionName;
 // NOTE: It is not safe to call this method while hasPendingViewRegistrations is YES.
 + (id)threadSpecialMessagesDatabaseView:(YapDatabaseReadTransaction *)transaction;
 
+// This method should be called _after_ all async database registrations have been started.
++ (void)asyncRegistrationCompletion;
```

Comment:
I guess it is confusing.  `TSStorageManager` kicks of the async registrations but doesn't wait  for them to complete, hence started.  I'm not sure how how to make these less clear, but IMO `completed` is more misleading.